WordPress/WordPressTest/ReaderSelectInterestsCoordinatorTests.swift

```diff
@@ -0,0 +1,169 @@
+import XCTest
+@testable import WordPress
+
+// MARK: - MockFollowedInterestsService
+class MockFollowedInterestsService: ReaderFollowedInterestsService {
```

Comment:
When I open a test the main thing I'm interested is in... the tests! :P

Here I have to scroll all the way down until I find 'em. Mocks can be after the main test suite, wdyt?

Reply:
Good idea. Pushed: https://github.com/wordpress-mobile/WordPress-iOS/pull/14476/commits/7af34cf571c6c2657a05df09d29d297b9e760c30